internal/version/service.go

```diff
@@ -30,61 +27,67 @@ type Config struct {
 
 // Service contains the information and variables to ensure the Software is up to date
 type Service struct {
+	log     *zap.Logger
 	config  Config
-	info    Info
 	service string
 
+	Info Info
+
 	Loop *sync2.Cycle
 
-	checked chan struct{}
```

Comment:
this is still needed for testplanet

Reply:
how so? nothing called IsUpToDate

Reply:
ah, right the check was disabled... nevermind, mind still foggy